### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Haroonabdulrazaq**
The project is focused on building responsive page

### CODE REVIEW

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# add the linters you want stickler to use for this project
```
Comment (lucasmazo32) [inline .stickler.yml:1]: - [x] Kindly enable the stickler in https://stickler-ci.com/ for this repo.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,857 @@
+<!DOCTYPE html>
```
Comment (lucasmazo32) [inline index.html:1]: - [x] Kindly fix the W3C warning.

> image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,857 @@
...
+
+    <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        Main Section
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+    <main class="main-section">
+      <div class="main-wrapper">
+        <div class="header-img hero-img-1 span-2-rows desk-header-img">
+          <div class="overlay">
+            <p class="img-text text-bg">
+              Netflix will cancel account if you havent used it in a while
```
Comment (lucasmazo32) [inline index.html:88]: - [x] Kindly add the logo to the main article for large screens. The second article doesn't have a gradient. Kindly fix that. 

- [x] For medium and small screens, the gradient should be there. The text isn't displayed for the articles with gradients and some text on the navbar is hidden. 

Source | Clone
-- | --
image | image
image | image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,857 @@
...
+          <p class="img-text text-bg-sm-3">What is Chademo? Let us explain</p>
+        </div>
+      </div>
+
+      <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        Latest News Section
+      ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+
+      <section class="latest-news">
+        <h2 class="heading2 orange">Latest News ></h2>
```
Comment (lucasmazo32) [inline index.html:108]: - [x] Kindly check the layout for the latest news section for medium devices.

Source | Clone
-- | --
image  | image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,857 @@
...
+          </div>
+        </div>
+      </section>
+      <!-- 
+      ======================================================================================
+      Funding Section
+      ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+
+      <section class="funding">
+        <h2 class="heading2 orange">Latest Funding Rounds on index.co ></h2>
```
Comment (lucasmazo32) [inline index.html]: - [x] Kindly fix the text layout for big and medium devices.

Source | Clone
-- | --
image | image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,857 @@
...
+                  >Canadian data start-up Cinchy lands $10m Series A on FinTech
+                  Futures</a
+                >
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        APP SECTION
```
Comment (lucasmazo32) [inline index.html:352]: - [x] The text in the images are not in the correct position for medium and small screens. Kindly fix that.

Source | Clone
-- | --
image | image

File: index.html [hunk 7]
```diff
@@ -0,0 +1,857 @@
...
+        </div>
+      </section>
+
+      <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        LATEST DEALS SECTION
+      +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+
+      <section class="latest-deals">
+        <div class="latest-deals-header">
+          <h2 class="heading2 orange inline-block">Latest Deals</h2>
```
Comment (lucasmazo32) [inline index.html:706]: - [x] Kindly check the images size for medium screens.

Source | Clone
-- | --
image | image

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good.

For the project to get approved, kindly implement the changes mentioned below.

### Happy coding! :computer:

## Lucas Mazo

File: index.html (hunk 7 above)
Comment (Haroonabdulrazaq) [reply index.html:706]: All changes have been fixed as suggested

**Comment by santiago-dev96** [comment]
# Pull Request under review by @santiago-rodrig (TSE)

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required 🚫

1. Your navigation bar is all messed up, please try to make it as similar as possible to the one that's in the original version.

### Yours

#### Large screens

Screen Shot 2020-06-01 at 7 20 15 AM

#### Medium screens

Screen Shot 2020-06-01 at 7 24 35 AM

#### Small screens

Screen Shot 2020-06-01 at 7 25 53 AM

### Original

#### Large screens

Screen Shot 2020-06-01 at 7 20 21 AM

#### Medium screens

Screen Shot 2020-06-01 at 7 24 43 AM

#### Small screens

Screen Shot 2020-06-01 at 7 25 43 AM

## Review process

I'm using a Mac, so I have the `open` command available, in Linux, it would be `xdg-open` instead, and in Windows, you could use some GUI app.

```zsh
git clone https://github.com/Haroonabdulrazaq/Responsive-Design.git &&
git checkout homepage &&
open index.html
```

## What to do now?

Please submit another CRR (Code Review Request) after implementing the required changes.

## How to reach out to me?

I'm available on slack as **srodrig**, happy coding! :wave:

**Review by trekab — CHANGES_REQUESTED** [review]
# Changes Required! :recycle: 
Hi @Haroonabdulrazaq and @talhawaqar, well done! There are a few more changes for you to make to complete this project. These are:
- [x] Your navigation breaks as soon as it hits the first breakpoint. The LOGO looks taller than the navbar and the rightmost navigation and search icon do not fully appear.
__Clone__
Clone 
__Original__
Original
- [x] The `Top 3 articles` section of your clone is too tall compared to the original page. Please resize it more appropriately.
- [x] The text in the `Top 3 articles` section should be at the bottom part of each article on large screens. They appear vertically centered on your clone.
- [x] On the `Categories (Apps, Gear, Tech, etc) section` the titles have too much space between them and their content on small screens. The text on the main images in this section should also always be at the bottom part of the images.
- [x] The `footer` should not change through the breakpoints.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Comment by Haroonabdulrazaq** [comment]
As for the last suggestion on footer the footer break on mobile, and that has been implmented

**Comment by christianotieno** [comment]
## Pull Request under review by Christian Otieno (TSE)

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @Haroonabdulrazaq  @talhawaqar  :wave:
-
You have done a splendid job implementing the above changes pointed out by the TSEs! :tada: Keep it up.
We, however, still, do have some few changes to carry out before we proceed (see review comment).
##
 -  [x] There appears an unwanted horizontal scroll during screen breakpoints. Kindly help fix this, thank you.

- [x] During small screen breakpoints, the navigation and sub-navigation row changes to a hamburger (bars) icon at the rightmost part (next to search icon) ([link to page](https://thenextweb.com/)).
- [x] Kindly take your time to compare and contrast between the two snapots and work on the differences. The idea here is to make our clone resemble the source page in almost every detail. The general look and feel are important. let's implement the missing features like dates, the font sizes and view the heights of our page's contents. We can ignore the disability icon and the countdown timer for now. Work on these parts and the footer as well (just the general height to march the original footers' height). See the differences below 👇 

|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/Haroonabdulrazaq/Responsive-Design/homepage/index.html)|
|:---:|:---:|
|tnw|tnw-clone|

Footer 

|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/Haroonabdulrazaq/Responsive-Design/homepage/index.html)|
|:---:|:---:|
|footer|footer-clone|

- [x] Don't forget to update the README with the current snapshot of your page once you have updated these changes.

- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Review by HADDADSOHAIB — APPROVED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
## Project Approved :+1: 

### Hello @Haroonabdulrazaq and @talhawaqar   :handshake: 

### Good job  your project is looking good :medal_sports: 

### You implemented all changes required by the previous TSEs, I approve this project.

**Pull request merged**